content/post/analytics_brain_trust.md

```diff
@@ -0,0 +1,86 @@
+---
+author: Scott Breitenother
+title: "Building your analytics brain trust"
+cover: "/img/analytics_brain_trust.jpg"
+publishDate: "2018-08-05"
+draft: true
+tags: 
+ - learning
+ - networking
+ - analytics
+ - brain trust
+---
+
+Imagine you hit a roadblock while trying to tackle a complex piece of analysis, using a python function or designing your first data organization. What do you do? Of course you first search the internet, but what do you do when you're really stuck? I like to phone a friend. 
+
```

Comment:
It may be useful to call out that this post is intended for building a network as oppose to "breaking in to Analytics". If this post includes the latter, then you might want to discuss topics like "learning about the field", "teaching yourself (stats / SQL / python) vs. taking online / night courses", "revamping your resume / linkedin to be more data-oriented", etc...

Reply:
Thanks, I think I might keep this out to keep the post streamlined.